### DISCUSSION TITLE
Cheat: Allow any path surface as queue

### DISCUSSION THREAD

**Discussion by karst** [OpenRCT2/OpenRCT2#18744] [Feature requests] [answered]
The game currently has a feature that is not possible without the scenery manager plugin, it's to use any kind of footpath surface as queue path. I think it'd be nice if there'd be a checkbox cheat to add all regular path surfaces to the list of queue paths as well. So you can just build it without the need of any plugin. 
![image](https://user-images.githubusercontent.com/1516893/205084104-4e715ca8-08a0-4caf-b511-f66dd9051d2b.png)

**Comment by Gymnasiast** [comment, marked as answer]
Implemented by https://github.com/OpenRCT2/OpenRCT2/pull/18844.

**Reply by karst** [reply]
Just realized that the PR to solve this is exactly 100 numbers higher. Kinda neat
